extensions/3DTILES_bounding_volume_S2/0.0.0/README.md

```diff
@@ -1,75 +1,92 @@
 
+<!-- omit in toc -->
 ## Contents
 
 - [3DTILES_bounding_volume_S2](#3dtiles_bounding_volume_s2)
-  - [Contributors](#contributors)
-  - [Status](#status)
-  - [Dependencies](#dependencies)
-  - [Optional vs. Required](#optional-vs-required)
-  - [Contents](#contents)
   - [Overview](#overview)
   - [Coordinate System](#coordinate-system)
-  - [Cell Token](#cell-token)
-  - [Heights](#heights)
+  - [Bounding Volume](#bounding-volume)
+    - [Cell Token](#cell-token)
+    - [Bounding Heights](#bounding-heights)
   - [Subdivision](#subdivision)
+  - [Property Reference](#property-reference)
   - [Schema Changes](#schema-changes)
 
 ## Overview
 
-[S2](http://s2geometry.io/) is a spherical geometry library that represents all data on a 3D sphere, unlike traditional libraries that use 2D planar projections. This makes it possible to represent the globe with no seams or singularities, with low distortion everywhere on Earth. The S2 library involves projecting the 6 faces of a cube on a unit sphere, creating 6 root "cells" that subdivide evenly into 4 tiles, into a quadtree structure.
+[S2](http://s2geometry.io/) is a spherical geometry library that maps the 6 faces of a unit cube onto the unit sphere. Typically, traditional GIS libraries use planar projections to map data between a 2D plane and a 3D ellipsoid (representing the Earth). Since a sphere is a closer approximation of the shape of the ellipsoid, S2 makes it possible to represent the globe with no seams or singularities, with low distortion everywhere on Earth.
 
-This extension to 3D Tiles enables using S2 cells as a `boundingVolume`, and uses `3DTILES_implict_tiling` to enforce subdivision.
+|Level 0|Level 1|
+|:-:|:-:|
+| ![Plane - Level 0](figures/plane_root.png) S2 cell (`"1"`) on the Earth cube| ![Plane - Level 1](figures/plane_children.png) Children of S2 cell (`"1"`) on the Earth cube |
+| ![Ellipsoid - Level 0](figures/ellipsoid_root.png)  S2 cell (`"1"`) on the WGS84 ellipsoid| ![Ellipsoid - Level 1](figures/ellipsoid_children.png) Children of S2 cell (`"1"`) on the WGS84 ellipsoid|
 
-| S2 Curve on Cube Face  |  S2 Curve on WGS84 Ellipsoid |
-|---|---|
+In S2, each face of the unit cube can be subdivided into 30 levels using a quadtree structure, in which each "cell" or tile on the grid subdivides into 4 equal cells or tiles at the subsequent level. This extension to 3D Tiles enables using these cells as the basis for bounding volumes for tiles.
+
+| S2 Curve on Earth cube |  S2 Curve on WGS84 ellipsoid |
+|:-:|:-:|
 | ![Math](figures/plane.png)  | ![Math](figures/ellipsoid.png)  |
 
+The S2 library uses a modified Hilbert curve to provide a one dimensional ordering of cells on the S2 Earth cube. This provides each cell, from level 1 to level 30, with a unique 64-bit identifier. Therefore, we can uniquely identify millimeter scale areas on Earth with their S2 cell IDs.
```

Comment:
I would use passive voice here instead of "we". Also the "Therefore" doesn't make sense in this sentence, it's not immediately obvious that 64-bit IDs imply mm precision

```suggestion
The S2 library uses a modified Hilbert curve to provide a one dimensional ordering of cells on the S2 Earth cube. This provides each cell, from level 1 to level 30, with a unique 64-bit identifier. Areas as small as a millimeter can be identified uniquely using S2 cell IDs.
```